Makefile

```
@@ -96,4 +96,4 @@ review:
 	if echo $$path | grep -q 'content/'; then path=`echo $$path | sed 's|content/||'`; fi; \
 	if [ -z "$$line_length" ]; then line_length=100; fi; \
 	export REVIEW=1; \
-	python tests/main.py --max-line-length=$$line_length $(SOURCE_DIR)/$$path
+	python3 tests/main.py --max-line-length=$$line_length $(SOURCE_DIR)/$$path
```

Make sure not to push up any unintended edits ;)
```suggestion
	python tests/main.py --max-line-length=$$line_length $(SOURCE_DIR)/$$path
```